algae_world/CMakeLists.txt

```diff
@@ -0,0 +1,218 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(smarc_stonefish_sims_algae_world)
```

Comment:
Probably rename it just algae_world, in line with the folder name as this is the convention. Also, smarc_stonefish_sims_algae_world is a bit of a moutful :).

Reply:
That's true :rofl:  I'll perhaps change the package and folder name to `algae_sim` for now :)